dbt_subprojects/hourly_spellbook/models/_sector/perpetual/projects/pika/optimism/pika_optimism_schema.yml

```
@@ -169,6 +169,51 @@ models:
       - *tx_to
       - *evt_index
       - *block_month
+    
+  - name: pika_v4_optimism_perpetual_trades
+    meta:
+      blockchain: optimism
+      sector: perpetual
+      project: pika_v4
+      contributors: princi
+    config:
+      tags: ['optimism', 'pika, pika_v4', 'perpetuals', 'perps']
+    description: >
+      Pika_v3 perpetuals trades/swaps on Optimism
+    data_tests:
+      - check_perpetuals_seed:
+            blockchain: optimism
+            project: Pika
+            version: 4
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
```

To match the simplified `unique_key`:
```suggestion
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - tx_hash
            - evt_index
```